packages/dashboard/src/images/horizontal bar char.svg

```diff
@@ -0,0 +1,22 @@
+<svg width="150" height="160" viewBox="0 0 150 160" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M145.369 160L23.8401 89.4623V2.67578L145.369 73.2138V160Z" fill="white"/>
+<path d="M149.999 157.311L145.369 159.997V73.2109L149.999 70.5234V157.311Z" fill="#D3D3D3"/>
+<path d="M145.369 73.2086L23.8405 2.67057L28.4413 0L149.97 70.538L145.369 73.2086Z" fill="#EFEFEC"/>
+<path opacity="0.1" d="M149.999 88.7175L145.369 91.4049V90.0703L149.999 87.3828V88.7175Z" fill="#010101"/>
+<path opacity="0.1" d="M145.369 91.4059L23.8401 20.8678V19.5332L145.369 90.0712V91.4059Z" fill="#010101"/>
+<path d="M79.8988 101.99L76.3213 104.076V94.5351L79.8988 92.4492V101.99Z" fill="#929292"/>
+<path d="M76.3215 104.074L0 59.5751V50.0605L76.3215 94.56V104.074Z" fill="#C2C2C2"/>
+<path d="M76.3222 94.5723L0.000736237 50.0729L3.55617 48L79.8772 92.4994L76.3222 94.5723Z" fill="#DADADA"/>
+<path d="M79.8988 122.841L76.3213 124.927V115.387L79.8988 113.301V122.841Z" fill="#003274"/>
+<path d="M76.3215 124.924L0 80.4247V70.9102L76.3215 115.41V124.924Z" fill="#00439B"/>
+<path d="M76.3217 115.422L0.000247955 70.9225L3.55568 68.8496L79.8767 113.349L76.3217 115.422Z" fill="#3369AF"/>
+<path d="M45.015 121.935L41.4375 124.021V114.48L45.015 112.395V121.935Z" fill="#003274"/>
+<path d="M41.4373 124.019L0 99.8661V90.3516L41.4373 114.504V124.019Z" fill="#00439B"/>
+<path d="M41.4375 114.517L0.000173569 90.3639L3.5556 88.291L44.9925 112.444L41.4375 114.517Z" fill="#3369AF"/>
+<path d="M100 134.914L96.4229 137V127.459L100 125.373V134.914Z" fill="#4A85B2"/>
+<path d="M96.4227 136.998L77.1121 125.821V116.307L96.4227 127.483V136.998Z" fill="#62B2ED"/>
+<path d="M96.4231 127.496L77.1124 116.319L80.6678 114.246L99.9785 125.423L96.4231 127.496Z" fill="#81C1F1"/>
+<path d="M65.1173 133.564L61.5398 135.65V126.109L65.1173 124.023V133.564Z" fill="#4A85B2"/>
+<path d="M61.5397 135.649L42.229 124.472V114.957L61.5397 126.134V135.649Z" fill="#0078D4"/>
+<path d="M61.54 126.144L42.2293 114.967L45.7848 112.895L65.0955 124.071L61.54 126.144Z" fill="#81C1F1"/>
+</svg>
```

Comment:
Do you think adding another folder inside images named AddCardPanelEmptyState and moving these svg's to that folder would be a good idea?Also the file names look a bit out of trend with the spaces between words could you make them CamelCase and check the typos of file names